Docs: Updated data links content (#27110)
grafana/grafana: docs/sources/linking/dashboard-links.md

# Dashboard links

> When you create a dashboard link, you can include the time range and current template variables to directly jump to the same context in another dashboard. This way, you don’t have to worry whether the person you send the link to is looking at the right data. For more information about specifying links, refer to [Data link variables]({{< relref "data-link-variables.md" >}}).

Reviewer: ```suggestion
When you create a dashboard link, you can include the time range and current template variables to directly jump to the same context in another dashboard. This way, you don’t have to worry whether the person you send the link to is looking at the right data. For other types of links, refer to [Data link variables]({{< relref "data-link-variables.md" >}}).
```

Dashboard links are their own things and not super related to data links. So changed this a bit to make that clear.
